README.md

```diff
@@ -5,11 +5,15 @@ This project contains an example server built with the Express framework written
 ## Quick Start
 
 - Use the Node version found in the .nvmrc file. This project uses `v21.1.0`.
-- Install the dependencies by running `pnpm install`. Here we use pnpm but you can install with a manager of your choice.
-- Rename [`config.example.json`](https://github.com/sednasystems/sedna-canvas-example/blob/main/config.example.json) in the root of the project to `config.json`.
-- To start the server in development, run `pnpm dev`. `npm dev` or `yarn dev` if you are using another package manager. This will start the Express server and `nodemon` will listen to any changes during development.
+- `pnpm install`
+- `mv config.example.json config.json`
+- Update `config.json` with correct app credentials.
 
-## Adding an App
+  If using workshop, these can be found by running `workshop/bin/create-canvas-app.sh` from the `services` repo
```

Comment:
Should we clarify what to do if you've already run this, or already run `setup-workshop --whatever-flag-you-add-to-enable-canvas`?

Reply:
actually this is what you do, because it won't create it'll just display - I think the fact that it's called `create-canvas-app` and used to list registered apps may be confusing - I'll think about renaming to `manage-canvas-apps` or something and let them do `manage-canvas-apps list` or `manage-canvas-apps create`